pkgs/servers/irc/charybdis/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, zlib, openssl, bison, flex }:
+
+stdenv.mkDerivation rec {
+  name = "charybdis-3.4.2-git22e4a9b";
+
+  src = fetchFromGitHub {
+    owner = "atheme";
+    repo = "charybdis";
+    rev = "22e4a9bc2b34915bf6e907e90d60bf7d002c7dc5";
+    sha256 = "0i22hc9al1g6ffa6vqw0lxdy1zj2y0ixgmmhi0kbc3j6xqp78kwj";
+  };
+
+  configureFlags = [
+    "--sysconfdir=/etc/charybdis"
+    "--localstatedir=/var"
+    "--disable-assert"
+    "--enable-openssl=${openssl}"
+    "--enable-ipv6"
+    "--with-program-prefix=charybdis-"
+  ];
+
+  buildInputs = [ zlib openssl bison flex ];
+
+  meta = {
+    description = "Charybdis IRC Daemon";
```

Comment:
Change to smth like `an extremely scalable ircd with some cooperation with the ratbox and ircu guys` (quote from their git repo). Omit dublication of a pkg name in a descriprion.